packages/liveblocks-client/src/room.ts

```diff
@@ -178,7 +178,11 @@ export type Effects = {
 };
 
 type Context = {
+  /**
+   * @deprecated Use `roomId` instead.
+   */
   room: string;
+  roomId: string;
```

Comment:
I read `context.room` somewhere, and assumed it was of type `room: Room`, but actually it was a `room: string` when I looked more closely, so changing the name. I'm duplicating the field here for now and deprecating it, but I'm not sure if that's even necessary. Seems to be a private field used only by the client itself?